docs/MSPX.md

```diff
@@ -19,33 +22,92 @@ mLRS provides the MspX technology, which is designed to improve the over-the-air
 
 ## INAV Configuration
 
-The baudrate should be set to 115200 as lower baudrates have shown irregular telemetry update rates on the radio due to the lack of flow control on the INAV side. 
-
-INAV 8.0 and later will also support a baudrate of 230400 but no change in performance or stability was noticed. Additionally with INAV 8.0 the performance of the serial link to a ground control station will be increased, due to some optimizations in INAV. 
+The baudrate should be set to 115200 or higher for consistent dataflow and RC control. Baudrates of less than 57600 can cause message loss on the INAV side and result in inconsistent RC control at 50Hz RC Rate, if MSP-RC is used (INAV Limitation).  
 
 To use a mLRS receiver with INAV in MspX mode, the following settings have to be applied in INAV:
 - Enable MSP for the Serial Port the mLRS receiver is connected to (UART 2 is recommended on most STM32 flight controllers; do not enable "Serial RX").
 - Set the baudrate to 115200 or higher.
+- INAV 8 and later will also support a baudrate of 230400 but no change in performance or stability was noticed.
   
 <img src="images/MSPX_ports.png" width="720px">
 
-- In the Receiver tab, select the Receiver Type in the Receiver Mode panel to MSP, and save settings.
+- In the Receiver tab, change the Receiver Mode to 'MSP' and the RSSI Source to 'MSP' (INAV 8 and later) then save settings.
   
-<img src="images/MSPX_receivermode.png" width="720px">
+<img src="images/MSPX_receiver.png" width="720px">
 
-- When connecting INAV 8.0 Configurator through mLRS it is highly recommended to enable the Wireless Mode switch before connection, for better link reliability (do not use Wireless Mode with versions earlier than 8.0, such as INAV 7.1).
+- When connecting INAV 8 Configurator through mLRS for flight monitoring, it is recommended to enable the Wireless Mode switch before connection, for better link reliability (do not use Wireless Mode with versions less than 8).
 
 <img src="images/MSPX_wirelessmode.png" width="360px">
 
 If your radio is connected, you should now be able to see the channel values update when you move the radio sticks. No further settings are needed and telemetry will work for EdgeTX/OpenTX radios after scanning for sensors.
+
+## Connection Types with different INAV Versions
+
+Depending on the chosen connection and serial mode, the functionality will slightly differ, especially in INAV versions before 8. This is a quick overview, what options are available and what functions to expect. 
+
+### INAV 8
+
+Serial Mode:
+- MspX
+  - provides telemetry sensors to your radio
+  - allows RC Control over MSP (rc override)
+  - Full MSP GCS Functionality with INAV Configurator, MWP and other applications
+  - comprehensive OSD and Blackbox Link Statistics with dedicated MSP messages provided by mLRS
+  - limited to 50Hz RC Rate (Only relevant for 2.4GHz FLRC Mode with 111Hz Packet rate, Will be limited to 37Hz RC rate)
```

Comment:
Lowercase for non acronyms, e.g.

'(only relevant for 2.4GHz FLRC mode, which will be limited to 37Hz RC rate)'

Reply:
Maybe mention that the datarate is not affected? 

'to 37Hz RC rate, but full datarate)'

Reply:
I think makes sense to leave as-is since this bullet is only talking about RC Rate.

Reply:
Makes sense. Set as requested.